symengine/lib/symengine_wrapper.pyx

```diff
@@ -4639,6 +4661,8 @@ IF HAVE_SYMENGINE_LLVM:
             addr2 = cast(<size_t>&self.lambda_double[0], c_void_p)
             return create_low_level_callable(self, addr1, addr2)
 
+    def llvm_loading_func(*args):
+        return LLVMDouble(args, _load=True)
```

Comment:
Actually I think the `_load` kwarg could go into `__cinit__` if we `cdef` the return value here:
```
def llvm_loading_func(*args):
    cdef LLVMDouble llvm_double = LLVMDouble(args, _load=True)
```

cf. e.g. [this](https://github.com/bjodah/symcxx/blob/master/symcxx/_symcxx.pyx.mako_template#L450)

Reply:
I'm not sure I follow. Can you elaborate more? (Or a PR. :smiley:)

Reply:
It's not important (actually not sure it matters much), so let's leave it for now (and I can try to make a PR against master after this is merged if I can find a way to hide the _load kwarg).